### ISSUE TITLE
Implement From<SystemTime> for FileTime

### ISSUE THREAD

**Issue by drrlvn** [alexcrichton/filetime#25]
This should be more ergonomic than using `filetime::FileTime::from_system_time()`.

**Comment by alexcrichton** [comment]
Certainly! This may have been added at a time where `SystemTime` was only recently stabilized, but I'd be totally up for adding such an impl today!
